### PR TITLE
Various church-related changes, again

### PR DESCRIPTION
**Pull request by 1N7Slayer**
## About The Pull Request

Final documentation of everything I've done so far:

- Adds Ravoxians (?ravoxites?) to the clergy table. Clerics, acolytes and priests can now be Ravoxian. Ravoxian clergy have no defined miracles - THIS IS BY DESIGN, they get stat and skill buffs as a trade-off to make them better kickass machines than their miracle-casting equivalents. As a consequence, they have no holy magic except the expected lesser miracle against the usual divine enemies.
- Tweaks some descriptions for the Divine patrons. Dendor is a force, not a son of Psydon. We also were requested not to use "Psydon" anymore.
- Fills out tiered spells to make it more rewarding for non-Astratan church roles to actually use their miracles. Greater heal is expected to be seen more often among certain acolytes.
- More open-ended outfitting for the priest, dependent on their chosen patron, rather than every non-Astratan priest getting an astrata amulet, for some reason. Some of the articles of clothing won't fit terribly, but this opens up the priest to be non-Astratan if they wish. Bear in mind this is only Astrata, Noc, Dendor, Eora, Ravox, Necra and Pestra currently. Xylix, Malum and Abyssor do not have features currently. I'll accept spritework to give the priest unique masks and robes per Divine, but right now this is just what works currently.
- Did I mention that Ravoxian clergy kick absolute ass?

## Why It's Good For The Game

After my previous nerf of the church roles, I felt it needed to add more features back into them for flexibility and diversity's sake.

## Pre-Merge Checklist

- [ ] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

File: code/datums/gods/patrons/divine_pantheon.dm [hunk 1]
```diff
@@ -58,6 +58,7 @@
 	t0 = /obj/effect/proc_holder/spell/invoked/lesser_heal
 	t1 = /obj/effect/proc_holder/spell/targeted/burialrite
 	t2 = /obj/effect/proc_holder/spell/targeted/churn
+	t3 = /obj/effect/proc_holder/spell/invoked/heal
 
 // ............ SAINTS ..................
 /datum/patron/divine/ravox
```
Comment (lavillastrangiato) [inline code/datums/gods/patrons/divine_pantheon.dm:64]: is this supposed to be pestra or is some part of this not showing up for me

**Comment by Cre77** [comment]
noc invisibility works now that stealth is re-enabled - i tested it extensively, just never got around to actually putting it back in

**Comment by 1N7Slayer** [comment]
> noc invisibility works now that stealth is re-enabled - i tested it extensively, just never got around to actually putting it back in

thanks king

**Comment by 1N7Slayer** [comment]
Final adjustments - a Ravoxian priest is an ABSOLUTE BADASS in combat.

**Comment by Optimism333** [comment]
I'm not a fan of opening up the priest role to other patrons, I think it waters down the premise of Astrata being the head goddess of the pantheon, with acolytes having more flexibility in patron selection. 

Setting aside my personal opinions on that however, I am even more against just giving the most powerful spell in the Astrata patron miracles to every priest, along with the spells from their patron. 

What incentive will there be to play an Astratan priest if a Noc priest can get a full roster of spells and still have highest tier Astrata  healing miracle? If it were added as a tier four spell to maybe one other patron that it made thematic sense for, that'd be less painful. But as a whole this just feels like a gutting of Astratan priests, their revival is no longer special, because every priest gets it, AND a full roster of four other spells.

It also plays into the idea that no matter what patron the priest selects, their jobs is to be a revive dispenser, which personally I'm not a fan of either.

**Comment by 1N7Slayer** [comment]
> I'm not a fan of opening up the priest role to other patrons, I think it waters down the premise of Astrata being the head goddess of the pantheon, with acolytes having more flexibility in patron selection. (Guayo disagrees though, so fine)
> 
> Setting aside my personal opinion on that however... I am even more against just giving the most powerful spell in the Astrata patron miracles (revival) to every priest, along with the spells from their patron.
> 
> What incentive will there be to play an Astratan priest if a Noc priest can get the same healing spells, revive, AND invisibility? If it were added as a tier four spell to maybe one other patron that it made thematic sense for, that'd be less painful. But as a whole this just feels like a gutting of Astratan priests, their revival is no longer special, because every priest gets it regardless of patron, AND a full roster of four other spells.
> 
> It also plays into the idea that no matter what patron the priest selects, their job is to be a revive dispenser, which personally I'm not a fan of either.

1. why does reviving need to be special to astratan clergy? because as you say in your insistence that the priest must be astratan, you are, whether consciously or not, still insisting that they be relegated to be the revivebot, because the status quo is that they are the only role (apart from astratan acolytes who grinded the devotion) who are capable of doing it apart from other astratan acolytes.
2. why does effective healing need to be special to astratan clergy? at present, pretty much all of the acolytes can do little besides feed water, stitch and spam the lesser miracle button, which isn't very good or effective healing compared to when an astratan acolyte does this with 2 healing buttons. because at present, all acolytes that are not astratan are less relevant compared to those who are from a mechanical standpoint: astratan acolytes are objectively superior with their miracle lineups contrasting to the others. now they are not, all acolytes have their healing and undead stuns but they also have their patron-specific miracles to make them stand out regardless.
3. why does the priest NEED to be astratan? it's already been covered that appropriate IC conduct as priest is to always venerate astrata, but your patron is your patron. because if we go off of your justification - very few people aside from the already established astratan priest characters will have any reason to go priest, because priest will simply "not be for them", due to the fact that astratan priests are expected to possess the tyrannical fanatical archetype of characterization. this has already led to repetition in roleplay where it's more or less just the same deal over and over. by contrast, with this slated change, this opens the priest up to have at least 7 times the amount and more of potential character concepts that one player could run with in regards to characterisation, personality and general dialogue writing.
4. everyone fulfills a mechanical purpose in game - that's how things get done and the round/narrative is able to move forward. revives are required as they stand in this game because dying to a stupid thing or mechanic is far too easy (see critical heartshots) -- so either revives are easy and effective or dying needs to be less easy and quick. there is not a middle ground to that situation, if you try to argue there is, im going to argue for the removal of the revive miracle entirely and we can see how the players enjoy this server with that experiment from that point onward.

**Comment by Optimism333** [comment]
Let's keep thing this discussion concise, if we can.

- I'm not addressing the lore/character point any further, I simply briefly recapped it. I know Guayo disagrees, that discussion is done as far as I'm concerned.
- I'd prefer to focus on the mechanical concern I outlined previously. As it stands in your pr, Astratan priests get four spells, one of which is revive? Yes? Priest picks Noc as their patron. They now get the same healing miracles, including revive, and their patron spells for a total of FIVE spells. In other words, you've made Noc priests as good as Astratan ones at healing, AND more flexible overall. You've made an Astratan priest less attractive to play because you've given their tier 4 spell to everyone to use for free. Even absent the argument about whether every priest should have revive, that's a problem.

**Comment by SELFHELL** [comment]
Just a preliminary review, but adding Ravox/Presta acolytes without unique sprites/outfits (and spells for ravox) doesnt match up with all the other Patrons with acolytes

**Comment by lavillastrangiato** [comment]
i just want the fucking pestra spells to work i dont give a shit if i have a unique outfit

**Comment by Optimism333** [comment]
> 1. Let Astrata have some specialization compared to the rest of the Patrons by keeping revival a spell exclusive to them. Gifting Anathema to every priest regarding of patron is a no. Clergy is thematic according to **their** patrons, and this is something you see on both their equipment, stats, and gear selection. If you see an Acolyte wearing a robe of Dendor, you're not gonna ask them for "heal plz" because they're not astratan. Every patron has their own domain, and Astrata's, by design, has always been healing. Keep it at that. I'm with Opti on this one, what's the point of playing an Astratan Priest if your biggest magic spell is available for everyone else?
> 2. The "people perceive church as medical" is a thing that ought to be culled and killed. People do not see the church as a healer, they see them as someone who can just click once on you so you don't have to heal wounds THE NATURAL WAY. WHICH IS STITCHING YOUR WOUNDS, AND RESTING IN A BED. Wah wah round removal I cannot play while asleep I love being afk etc etc, but there are other ways. If there's no Astratans, heal through potions, feldsher, and fucking sleep. Let people rely less on healbots.
> 3. As also approved by Loreholder Guayo (and I agree), let Priest be open to any God, Force, or Saint they wish to worship, but still **do not give them the revival spell** just because they're priest. Also, make sure they have different robes or a distinct physical appearance depending on what they worship, to make people react more easily towards them. I understand that a "gameplay loop" is needed, but to be honest this chalks out more to player lazyness than an actual LACK of OPTIONS AND MECHANICS to circumvent a lack of healing magic. It's been long since we were no longer dependant on magic to heal wounds, and this is a good thing both mechanically and lore-wise, so we shouldn't measure our options with the scope of what people HAD been used to.

I can live with the compromise of opening the priest slot to all patrons (begrudgingly), if the revival spell won't be given to every priest for free.

**Comment by BadAtThisGame302** [comment]
> > 1. Let Astrata have some specialization compared to the rest of the Patrons by keeping revival a spell exclusive to them. Gifting Anathema to every priest regarding of patron is a no. Clergy is thematic according to **their** patrons, and this is something you see on both their equipment, stats, and gear selection. If you see an Acolyte wearing a robe of Dendor, you're not gonna ask them for "heal plz" because they're not astratan. Every patron has their own domain, and Astrata's, by design, has always been healing. Keep it at that. I'm with Opti on this one, what's the point of playing an Astratan Priest if your biggest magic spell is available for everyone else?
> > 2. The "people perceive church as medical" is a thing that ought to be culled and killed. People do not see the church as a healer, they see them as someone who can just click once on you so you don't have to heal wounds THE NATURAL WAY. WHICH IS STITCHING YOUR WOUNDS, AND RESTING IN A BED. Wah wah round removal I cannot play while asleep I love being afk etc etc, but there are other ways. If there's no Astratans, heal through potions, feldsher, and fucking sleep. Let people rely less on healbots.
> > 3. As also approved by Loreholder Guayo (and I agree), let Priest be open to any God, Force, or Saint they wish to worship, but still **do not give them the revival spell** just because they're priest. Also, make sure they have different robes or a distinct physical appearance depending on what they worship, to make people react more easily towards them. I understand that a "gameplay loop" is needed, but to be honest this chalks out more to player lazyness than an actual LACK of OPTIONS AND MECHANICS to circumvent a lack of healing magic. It's been long since we were no longer dependant on magic to heal wounds, and this is a good thing both mechanically and lore-wise, so we shouldn't measure our options with the scope of what people HAD been used to.
> 
> I could live with the compromise of opening the priest slot to all patrons (begrudgingly), if the revival spell won't be given to every priest for free.

Then people will shit on a Priest that doesn't go and be Astratan so they can be easy revives, since that has happened before especially due to the current situation that is completely bugged and doesn't force your patron to another.

**Comment by Optimism333** [comment]
> > > 1. Let Astrata have some specialization compared to the rest of the Patrons by keeping revival a spell exclusive to them. Gifting Anathema to every priest regarding of patron is a no. Clergy is thematic according to **their** patrons, and this is something you see on both their equipment, stats, and gear selection. If you see an Acolyte wearing a robe of Dendor, you're not gonna ask them for "heal plz" because they're not astratan. Every patron has their own domain, and Astrata's, by design, has always been healing. Keep it at that. I'm with Opti on this one, what's the point of playing an Astratan Priest if your biggest magic spell is available for everyone else?
> > > 2. The "people perceive church as medical" is a thing that ought to be culled and killed. People do not see the church as a healer, they see them as someone who can just click once on you so you don't have to heal wounds THE NATURAL WAY. WHICH IS STITCHING YOUR WOUNDS, AND RESTING IN A BED. Wah wah round removal I cannot play while asleep I love being afk etc etc, but there are other ways. If there's no Astratans, heal through potions, feldsher, and fucking sleep. Let people rely less on healbots.
> > > 3. As also approved by Loreholder Guayo (and I agree), let Priest be open to any God, Force, or Saint they wish to worship, but still **do not give them the revival spell** just because they're priest. Also, make sure they have different robes or a distinct physical appearance depending on what they worship, to make people react more easily towards them. I understand that a "gameplay loop" is needed, but to be honest this chalks out more to player lazyness than an actual LACK of OPTIONS AND MECHANICS to circumvent a lack of healing magic. It's been long since we were no longer dependant on magic to heal wounds, and this is a good thing both mechanically and lore-wise, so we shouldn't measure our options with the scope of what people HAD been used to.
> > 
> > 
> > I could live with the compromise of opening the priest slot to all patrons (begrudgingly), if the revival spell won't be given to every priest for free.
> 
> Then people will shit on a Priest that doesn't go and be Astratan so they can be easy revives, since that has happened before especially due to the current situation that is completely bugged and doesn't force your patron to another.

Well, then there's three possibilities I would lay out:
- Leave the priest locked to Astratan, fix the late join bug that allows you to join as another patron
- Unlock priest patrons, give all priests revive, give the Astrata spell list a new t3 spell so Astratan priests don't fall behind
- Unlock priest patrons, leave only Astrata priests with revive (maybe one other patron at most), and just ignore that quasi-OOC grudge behavior because players who treat the church like a revive dispensary and get mad when it's not are getting angry when something in a round deviates from an optimized meta. And that's not a good attitude to cater towards, imo

Personally I would favor option 1, though I would be open to option 3 as a compromise. I would rather that revives and the ability to revive remain rare and be given a more tangible cost in the future, than anything else.

**Pull request closed without merging**